src/providers/postgres/qgspgnewconnection.cpp

```diff
@@ -82,6 +82,7 @@ QgsPgNewConnection::QgsPgNewConnection( QWidget *parent, const QString &connName
     cb_geometryColumnsOnly->setChecked( settings.value( key + "/geometryColumnsOnly", true ).toBool() );
     cb_dontResolveType->setChecked( settings.value( key + "/dontResolveType", false ).toBool() );
     cb_allowGeometrylessTables->setChecked( settings.value( key + "/allowGeometrylessTables", false ).toBool() );
+    cb_showRasterOverviews->setChecked( settings.value( key + "/showRasterOverviews", true ).toBool() );
```

Comment:
Let's have this `false` by default for new connections.

We should only default to `true` for _existing_ connections, which is properly handled later on.

Reply:
fixed